emantomo/requirements/Missing_Dependencies

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
What is this file for?

Reply:
This file is used to report possible solutions to errors that may occur during the installation of Eman to help users. Since this is not so documented on the Eman installation page if we find any dependency that is usually leading to installation errors we can write them here so they appear in a way similar to Xmipp.

Reply:
DO we need to have it in the source control? or should we ignore it?

Reply:
We would need to add it as part of the Plugin when we release it so it can be accessed during the installation if needed. I am not sure if this implies we have to have it in the source control or if we can add it to the .gitignore file.